<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>549</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Transfer Data</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="host_label">
      <property name="maximumSize">
       <size>
        <width>382</width>
        <height>15</height>
       </size>
      </property>
      <property name="text">
       <string>Target IP:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="host_input">
      <property name="maximumSize">
       <size>
        <width>382</width>
        <height>22</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="username_label">
      <property name="maximumSize">
       <size>
        <width>382</width>
        <height>15</height>
       </size>
      </property>
      <property name="text">
       <string>Target Username:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="username_input">
      <property name="maximumSize">
       <size>
        <width>382</width>
        <height>22</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="private_key_pass_label">
      <property name="maximumSize">
       <size>
        <width>382</width>
        <height>15</height>
       </size>
      </property>
      <property name="text">
       <string>Target Password:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="private_key_pass_input">
      <property name="maximumSize">
       <size>
        <width>382</width>
        <height>22</height>
       </size>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="private_key_dir_label">
      <property name="maximumSize">
       <size>
        <width>382</width>
        <height>15</height>
       </size>
      </property>
      <property name="text">
       <string>Private Key Directory:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="private_key_dir_input">
      <property name="maximumSize">
       <size>
        <width>382</width>
        <height>22</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="private_key_dir_button">
      <property name="maximumSize">
       <size>
        <width>382</width>
        <height>23</height>
       </size>
      </property>
      <property name="text">
       <string>Browse</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="ignore_host_key_checkbox">
      <property name="maximumSize">
       <size>
        <width>382</width>
        <height>21</height>
       </size>
      </property>
      <property name="text">
       <string>Ignore Host Key</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="source_dir_label">
      <property name="maximumSize">
       <size>
        <width>382</width>
        <height>15</height>
       </size>
      </property>
      <property name="text">
       <string>Source Directory:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="source_dir_input">
      <property name="maximumSize">
       <size>
        <width>382</width>
        <height>22</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="source_dir_button">
      <property name="maximumSize">
       <size>
        <width>382</width>
        <height>23</height>
       </size>
      </property>
      <property name="text">
       <string>Browse</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="target_dir_label">
      <property name="maximumSize">
       <size>
        <width>382</width>
        <height>15</height>
       </size>
      </property>
      <property name="text">
       <string>Target Directory:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="target_dir_input">
      <property name="maximumSize">
       <size>
        <width>382</width>
        <height>22</height>
       </size>
      </property>
      <property name="text">
       <string>/home/</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="save_config_button">
      <property name="maximumSize">
       <size>
        <width>382</width>
        <height>23</height>
       </size>
      </property>
      <property name="text">
       <string>Save Config</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="load_config_button">
      <property name="maximumSize">
       <size>
        <width>382</width>
        <height>23</height>
       </size>
      </property>
      <property name="text">
       <string>Load Config</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="clear_config_button">
      <property name="maximumSize">
       <size>
        <width>382</width>
        <height>23</height>
       </size>
      </property>
      <property name="text">
       <string>Clear Config</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="upload_button">
      <property name="maximumSize">
       <size>
        <width>382</width>
        <height>23</height>
       </size>
      </property>
      <property name="text">
       <string>Upload</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile_Transfer">
    <property name="title">
     <string>File Transfer</string>
    </property>
   </widget>
   <addaction name="menuFile_Transfer"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
 <slots>
  <slot>testPrint()</slot>
 </slots>
</ui>
